docs/source/using/addons.adoc

```diff
@@ -100,53 +95,52 @@ $ openshift admin ca create-server-cert \
   --key=/etc/secrets/registry.key
 ----
 
-<<table-supported-addon-variables>> shows the currently supported variables
-which are available for interpolation.
-
 [[table-supported-addon-variables]]
 .Supported add-on variables
+
```

Comment:
The comment is the same as for the example ones. You are pulling text into the table, similar as to the examples above.